src/console_node/consoleNodeMain.go

```diff
@@ -87,6 +88,31 @@ func setPodName() {
 	conAggLogFile = conAggLogFileBase + podName + ".log"
 }
 
+// identify where the current pod is running
+func setPodLocation(os OperatorService) {
+	var resp *PodLocationDataResponse
+	var err error
+	var retryCounter int = os.MaxOperatorRetries()
+	var retryInterval time.Duration = os.OperatorRetryInterval()
+
+	for i := 0; i < os.MaxOperatorRetries(); i++ {
+		resp, err = os.getPodLocation(podName)
+		if err != nil {
+			log.Printf("Error: Failed to retrieve location from console-operator, retrying in %f for %d more retries\n", retryInterval.Seconds(), retryCounter)
+			retryCounter--
+		} else {
+			podLocData = resp
+			return
+		}
+
+		// Block and retry until location is returned
+		time.Sleep(retryInterval)
+	}
+
+	// TODO: Should we panic here or just let non-filtered pass to the console-data
+	log.Panicf("Error: Failed to retrieve location from console-operator")
```

Comment:
A different service failing to respond is no reason to crash this one. You get an entire cascade of failing services that way. Better to log the failure to connect and try in an infinite loop.